exporter/stats/stackdriver/stackdriver_test.go

```diff
@@ -35,7 +36,13 @@ func TestExporter_makeReq(t *testing.T) {
 	}
 	defer stats.DeleteMeasure(m)
 
-	cumView, err := stats.NewView("cumview", "desc", nil, m, stats.CountAggregation{}, stats.Cumulative{})
+	key, err := tag.NewKey("test-key")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer tag.Delete(key)
```

Comment:
This is not deleting the key but returning a mutator to delete a value keyed with key.

(FWIW, I think the naming for Insert, Upsert, Delete, etc is not great and creates confusion.)

Reply:
Agree, this method is a little confusing to me at first. Removed.